src/main.ts

```diff
@@ -1,16 +1,19 @@
-import * as core from '@actions/core';
-import { ToolsDirectory } from './toolsDirectory';
-import * as dotnet from './dotnet';
-import * as cake from './cake';
-import { CakeArgument } from './cakeParameter';
+import * as core from "@actions/core";
+import { ToolsDirectory } from "./toolsDirectory";
+import * as dotnet from "./dotnet";
+import * as cake from "./cake";
+import { CakeArgument } from "./cakeParameter";
```

Comment:
The convention in the codebase is to use single quotes. Please, remove this change.

```suggestion
import * as core from '@actions/core';
import { ToolsDirectory } from './toolsDirectory';
import * as dotnet from './dotnet';
import * as cake from './cake';
import { CakeArgument } from './cakeParameter';
```